redbot/cogs/filter/filter.py

```diff
@@ -150,14 +152,35 @@ async def _filter(self, ctx: commands.Context):
         """
         pass
 
+    @_filter.command(name="clear")
+    async def _filter_clear(self, ctx: commands.Context):
+        """Clears this servers filter list."""
+        server = ctx.guild
+        author = ctx.author
+        filter_list = await self.config.guild(server).filter()
+        if not filter_list:
+            return await ctx.send(_("The filter list for this server is empty."))
+        await ctx.send(_("Are you sure you want to clear this server's filter list? (yes/no)"))
```

Comment:
"yes/no" is not conditional on the used language so should not be part of the translated text:
```suggestion
        await ctx.send(_("Are you sure you want to clear this server's filter list?") + " yes/no")
```